test/externalTests/euler.sh

```diff
@@ -66,6 +66,10 @@ function euler_test
     # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/2453 gets fixed.
     sed -i "/expectError: 'JUNK_UPGRADE_TEST_FAILURE'/d" test/moduleUpgrade.js
 
+    # Workaround to make euler test pass
+    # TODO: Remove this when https://github.com/euler-xyz/euler-contracts/issues/119 gets fixed.
+    sed -i 's/domainSeparatorMatch).to.equal(true/domainSeparatorMatch).to.equal(false/g' test/eulStakes.js
```

Comment:
You need to escape `.` to match actual dots rather than any character:
```suggestion
    sed -i 's/domainSeparatorMatch)\.to\.equal(true/domainSeparatorMatch)\.to\.equal(false/g' test/eulStakes.js
```
Probably won't make a difference here but still, better to keep it correct.